jobs/mysql/spec

```diff
@@ -247,3 +247,12 @@ properties:
   cf_mysql.mysql.enable_galera:
     default: true
     description: 'Advanced feature. Only use this when deploying a single MySQL node with no intention to run any other components of cf-mysql-release.'
+
+  cf_mysql.mysql.long_query_time:
```

Comment:
consider renaming "long_query_time" into "slow_query_time" for consistency with existing user-facing wording in the release

Reply:
The MySQL variable is  **long_query_time** and not **slow_query_time**
```
+-----------------+----------+
| Variable_name   | Value    |
+-----------------+----------+
| long_query_time | 1.000000 |
+-----------------+----------+

```

Reply:
I understand your point about also keeping consistency with original mysql cnf parameter naming. No strong opinion there. Please ignore this comment then.